_posts/2017-08-21-elasticsearch-loader-0.10.0-released/index.md

```diff
@@ -0,0 +1,69 @@
+
+In this post, we will cover: 
+
+1. [NSQ support](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#nsq-support)
+2. [Support for writing raw JSONs](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#json)
+3. [Support for "AT_TIMESTAMP" as initial position](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#at-timestamp)
+4. [Configuration changes](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#config)
+5. [Contributing](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#contributing)
+
+<!--more-->
+
+<h2 id="nsq-support">1. NSQ Support</h2>
+
+With this release, we are adding support for [NSQ][nsq-website] as an event source: the loader can now sink Snowplow enriched events from an NSQ topic to Elasticsearch. 
+
+[NSQ][nsq-website] is a real-time distributed messaging platform. For more information on getting started, read the [NSQ installation guide][nsq-installation]. We are planning on migrating Snowplow Mini to use NSQ under the hood, and so this new functionality is a stepping stone to this goal.
+
+Assuming you have NSQ set up already, you will need to make some changes to the Elasticsearch Loader's configuration file:
+
+* Change the "source" field's value to "nsq"
+* Complete the NSQ section of the config per your NSQ setup
+* Change the stream names to match your NSQ topic names
+
+<h2 id="json">2. Support for writing raw JSONs</h2>
+
+We're keen to widen the community using the Snowplow loaders, and in support of this we have added the ability to write non-Snowplow JSON payloads to Elasticsearch.
+
+You can think of this as an open-source version of [Amazon Kinesis Firehose][kinesis-firehose], but much more flexible, working as it does with NSQ, Kinesis and stdin sources and non-Elasticsearch Service clusters.
+
+To write non-Snowplow JSONs to Elasticsearch, just change `enabled` field to "plain-json" in the [config file][example-config].
+
+<h2 id="at-timestamp">3. Support for "AT_TIMESTAMP" as initial position</h2>
+
+Prior to this release, there were two possible configuration options to serve as the initial processing position for the Kinesis stream: `TRIM_HORIZON` and `LATEST`.
+
+These determine what happens on the first run of the application, when the Kinesis Connectors Library inside the Elasticsearch Loader creates a DynamoDB table to track what it has consumed from the stream so far. Either on first run it would start consuming from `LATEST` (the most recent record in the stream), or from `TRIM_HORIZON` (the oldest record available in the stream).
+
+This release adds a third option for the initial position, `AT_TIMESTAMP`. With the `AT_TIMESTAMP` option, consuming will start from the specified timestamp. You can get more information about initial positions from [our own guide to Kinesis at-least-once processing][kinesis-at-least-once-processing].
```

Comment:
It'd be cool to add a sentence on how to leverage this in the configuration and mention the date format